main.cpp

```diff
@@ -1,52 +1,104 @@
 #include <iostream>
 #include <fstream>
 #include <string>
+#include <sstream>  //convert
+#include <cstring>  //strcmp
+#include <functional> //hash
 
-bool test_mode = true;
+namespace colors
+{
+    char red[]  = "\033[31m";
+    char blue[] = "\033[34m";
+    char none[] = "\033[0m";
+}
+
+bool test_mode = false;
```

Comment:
It is better to use constexpr In such case (hope you do not want to change the mode in the middle of the program execution)